kubernetes/debug.yml

```diff
@@ -47,6 +47,8 @@ spec:
         cpu: '0.5'
         memory: '300Mi'
     command: ["sleep", "9999"]
+    securityContext:
+      runAsUser: 0
```

Comment:
Why does it need root?

Reply:
so you can install/change while debugging